db/seeds.rb

```diff
@@ -5,35 +5,34 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
-h1 = Hall.create(name: 'Hall 1', capacity: 20)
-h2 = Hall.create(name: 'Hall 2', capacity: 50)
-h3 = Hall.create(name: 'Hall 3', capacity: 50)
-h4 = Hall.create(name: 'Hall 4', capacity: 50)
-h5 = Hall.create(name: 'Hall 5', capacity: 50)
-h6 = Hall.create(name: 'Hall 6', capacity: 100)
-h7 = Hall.create(name: 'Hall 7', capacity: 100)
-h8 = Hall.create(name: 'Hall 8', capacity: 100)
-h9 = Hall.create(name: 'Hall 9', capacity: 100)
-h10 = Hall.create(name: 'Hall 10', capacity: 200)
+Hall.find_or_create_by(:name => 'Hall 1', :capacity => 20)
```

Comment:
the syntax `{:name => 'Hall 1' }` is old and we prefer to use new one `{ name: 'Hall 1' }`